### ISSUE TITLE
More results than required

### ISSUE THREAD

**Issue by MatlabRes** [Raboro/Obsidian-Birthday-Tracker-Plugin#1]
Hello

I have this text in md  file:
```
name=me; birthday=22/12/1990
name=Hans Ap; birthday=01/05/2004
name=Mats Mattis; birthday=21/03/1999
```

![Screenshot_20230829_130155](https://github.com/Raboro/Obsidian-Birthday-Tracker-Plugin/assets/22655314/711fd11f-824b-412c-b435-bc86e6f90ea8)

But have 5 results! Where 2 empty. 

P.S. It good idea for plugin. But need render block to home.md page the next few points. 

Best use dataview.
 example:

```dataview
TABLE Name, date(BirthDay, "dd.MM.yyyy")  as "BirtchDay"
FROM "Peoples"
WHERE (date(BirthDay, "dd.MM.yyyy").month - date(today).month <= 1 AND date(BirtchDay, "dd.MM.yyyy").month - date(today).month >= 0) OR date(BirthDay, "dd.MM.yyyy").month - date(today).month <= -10
SORT date(BirthDay, "dd.MM.yyyy").Day DESCENDING
SORT date(BirthDay, "dd.MM.yyyy").Month DESCENDING
LIMIT 3
```

**Comment by Raboro** [comment]
Thanks for the issue.
I tested it and when I only add this it worked fine:
name=me; birthday=22/12/1990
name=Hans Ap; birthday=01/05/2004
name=Mats Mattis; birthday=21/03/1999

But when I added the \``` before and after than the problem occured. 
So just remove the ``` before and after your birthdays. (Remove the code block):
Intead of this:
```
name=me; birthday=22/12/1990
name=Hans Ap; birthday=01/05/2004
name=Mats Mattis; birthday=21/03/1999
```

do this:
name=me; birthday=22/12/1990
name=Hans Ap; birthday=01/05/2004
name=Mats Mattis; birthday=21/03/1999

**Comment by Raboro** [comment]
Included a check, your text is now working. (Included in the latest release [1.0.4](https://github.com/Raboro/Obsidian-Birthday-Tracker-Plugin/releases/tag/1.0.4))
